tools/tuf/releaser.sh

```diff
@@ -173,13 +173,6 @@ release_fleetd_to_edge () {
     echo "Releasing fleetd to edge..."
     ORBIT_TAG="orbit-v$VERSION"
     prompt "A tag will be pushed to trigger a Github Action to build desktop and orbit."
-    pushd "$GIT_REPOSITORY_DIRECTORY"
-    git tag "$ORBIT_TAG"
-    git push origin "$ORBIT_TAG"
-    if [[ "$SKIP_PR" != "1" ]]; then
-        create_fleetd_release_pr
-    fi
-    popd
```

Comment:
Running this step as a separate step/action because `create_fleetd_release_pr` involves creating a PR against `main` which usually has conflicts when releasing patches (which then makes the whole run fail)